Makefile.local

```diff
@@ -38,6 +38,6 @@ test-upload:
 upload:
 	twine upload dist/*
 
-.PHONY: dist-tools dist test-upload upload
+.PHONY: build-tools build test-upload upload
```

Comment:
I usually follow a pattern like:

```make
.PHONY: target1
target1: ...
        ...

.PHONY: target2
target2: ...
        ...
```

Up to you, but it makes it much less likely these ever go out of sync again.

Reply:
Fixed that in #150